new_platforms/vscode-extension/README.md

```diff
@@ -142,12 +153,28 @@ data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false
 ## Known Issues
 
 * Building SGX enclaves is not currently supported.
+* On Linux, if you see errors in an Edge container build that involve `standard_init_linux.go:207`, you are likely cross-building a container.  Adding these packages solved the cross-build issue on Ubuntu 19.04:
```

Comment:
Consider changing the anecdotal "solved" to a factual "solves".

Reply:
heh ... the anecdotal reflected my confidence.  if you think 'solves' is appropriate, i'll change it.

Reply:
Same comment here and throughout.